contrib/demos-unmaintained/demo/prototype2-script/user-rbac.yaml

```diff
@@ -7,7 +7,7 @@ rules:
 - apiGroups: ["tenancy.kcp.dev"]
   resources: ["clusterworkspaces/content"]
   resourceNames: ["demo"]
-  verbs: ["access","member"]
```

Comment:
next time we should acknowledge the smell of calling a noun, a verb ;)

Reply:
lol, yes :-) in fact, i do believe i found precendences in k8s using nouns as well :-|